src/configuration/applier/hostescalation.cc

```diff
@@ -231,14 +231,6 @@ void applier::hostescalation::remove_object(
         break;
       }
     }
-    // Notify event broker.
-    timeval tv(get_broker_timestamp(nullptr));
-    broker_adaptive_escalation_data(NEBTYPE_HOSTESCALATION_DELETE,
-                                    NEBFLAG_NONE,
-                                    NEBATTR_NONE,
-                                    it->second.get(),
-                                    &tv);
-
```

Comment:
does this code is still call elsewhere ?

Reply:
in src/hostescalation